wiki/Tournaments/UKCC/UKCC7/en.md

```diff
@@ -0,0 +1,580 @@
+| 81-96 | ![][flag_GB] [_Snowy](https://osu.ppy.sh/users/10710046), ![][flag_GB] [Tesco Meal Deal](https://osu.ppy.sh/users/6471972), ![][flag_GB] [DeltaZero](https://osu.ppy.sh/users/6472042), ![][flag_GB] [JO3-M4K](https://osu.ppy.sh/users/7659789), ![][flag_GB] [Felicia](https://osu.ppy.sh/users/6177798), ![][flag_GB] [SS_Eclipse](https://osu.ppy.sh/users/9468006), ![][flag_GB] [Sith](https://osu.ppy.sh/users/8112797), ![][flag_GB] [JackRB](https://osu.ppy.sh/users/10007343), ![][flag_GB] [lokkut](https://osu.ppy.sh/users/4776591), ![][flag_GB] [Asda Meal Deal](https://osu.ppy.sh/users/6749950), ![][flag_GB] [PsychoStatic](https://osu.ppy.sh/users/7291594), ![][flag_GB] [smh my head](https://osu.ppy.sh/users/12203874), ![][flag_GB] [Phymiath](https://osu.ppy.sh/users/9106403), ![][flag_GB] [Elit3](https://osu.ppy.sh/users/10086758), ![][flag_GB] [Blukey](https://osu.ppy.sh/users/7813034), ![][flag_GB] [GongoTheGecko](https://osu.ppy.sh/users/9227012) |
+| 97-111 | ![][flag_GB] [Damarsh](https://osu.ppy.sh/users/7465147), ![][flag_GB] [DrexiN](https://osu.ppy.sh/users/5363646), ![][flag_GB] [cfman29](https://osu.ppy.sh/users/9023597), ![][flag_GB] [God](https://osu.ppy.sh/users/10502908), ![][flag_GB] [Mantle](https://osu.ppy.sh/users/7225966), ![][flag_GB] [aroS](https://osu.ppy.sh/users/7193474), ![][flag_IE] [Neverslain](https://osu.ppy.sh/users/10644912), ![][flag_IE] [kaniciwah](https://osu.ppy.sh/users/9371954), ![][flag_GB] [josph12](https://osu.ppy.sh/users/8584840), ![][flag_GB] [mamoopy](https://osu.ppy.sh/users/9384951), ![][flag_GB] [fadentic](https://osu.ppy.sh/users/13453905), ![][flag_GB] [Metzo](https://osu.ppy.sh/users/10633982), ![][flag_GB] [Bowris](https://osu.ppy.sh/users/8562850), ![][flag_GB] [_Phazy](https://osu.ppy.sh/users/3956707), ![][flag_IE] [Broznix](https://osu.ppy.sh/users/13909259) |
+
+---
+
+## Podium
+
+UKCC 7 concluded on the 27th October 2019.
+
+| Placing | Player |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "Winner") | [Bubbleman](https://osu.ppy.sh/users/5182050) |
+| ![Silver crown](/wiki/shared/crown-silver.png "Second place") | [Spare](https://osu.ppy.sh/users/2204373) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "Third place") | [AJT](https://osu.ppy.sh/users/3181083) |
+
+---
+
+## Map pools
+
+![Map pools](img/mappool.png)
+
+*NB - Map packs may no longer be available due to age.*
+
+### Losers Final + Grand Final
+
+**[Download the map pack here! (148.0MB)](https://mega.nz/#!TmhgUK5K!7kkYZPdolGvJbuCezRBj1SFJe6J-qZRu0B4qXTwXwzU)**
+
+- Nomod
```

Comment:
Indent following parent. (i think it's 2 instead of 4 or just follow the parent)